test/resources/utils.go

```diff
@@ -97,6 +102,26 @@ func VerifySuccessfulAzVolumeAttachments(pod PodDetails, azDiskClient *azDiskCli
 	return
 }
 
+// Execute the command for all pods in the namespace, looking for expectedString in stdout
+func pollForStringInPodsExec(namespace string, pods []string, command []string, expectedString string) {
+	for _, pod := range pods {
+		args := append([]string{"exec", pod, "--"}, command...)
+		err := wait.PollImmediate(testconsts.Poll, pollForStringTimeout, func() (bool, error) {
+			stdout, err := framework.RunKubectl(namespace, args...)
```

Comment:
Can we do this asynchronously and then wait for the results from all the pods through a buffered channel?

Reply:
Good idea!